.editorconfig

```diff
@@ -10,6 +10,7 @@ root = true
 insert_final_newline = true
 indent_style = space
 indent_size = 4
+trim_trailing_whitespace = true
```

Comment:
This is often problematic. I would not recommend using it.

Reply:
What kind of problems do this introduce?

@wli3, Are you okay if there will start to be differences than dotnet/runtime?

Reply:
* Some files have significant trailing whitespace (e.g. Markdown, C# verbatim string literals)
* Some files have generated trailing whitespace (e.g. the .Designer.cs files created from a .resx file)

Text editors ignore these in some cases but not others, so you end up with the potential to make incorrect changes and/or unintentional code churn.

Reply:
These are really interesting scenarios that I never thought of.

I agree that this option should be removed, but I'll wait for @wli3

Reply:
i refer removing that

Reply:
```suggestion
```